spec/support/features/session_helper.rb

```diff
@@ -162,7 +162,7 @@ def acknowledge_and_confirm_recovery_code
       click_on button_text, class: 'recovery-code-continue'
 
       code_words.size.times do |index|
-        fill_in "recovery-#{index}", with: code_words[index].downcase
+        fill_in "recovery-#{index}", with: code_words[index].downcase + 'xxxx'
```

Comment:
We do check minimum length and maximum length. Password must be between 8 and 128 characters: https://github.com/18F/identity-idp/blob/master/app/validators/form_password_validator.rb#L10

For recovery code, we want to verify that it is exactly 4*recovery_code_length characters since that's the only length allowed, right?